api/handler/put.go

```diff
@@ -168,6 +168,9 @@ const (
 	basicACLReadOnly  = "public-read"
 	basicACLPublic    = "public-read-write"
 	cannedACLAuthRead = "authenticated-read"
+
+	amzBucketOwnerField    = "x-amz-object-ownership"
```

Comment:
Can't this collide with some real header? I'd expect a bit more magic value here. Although if the rule is always the last one (after default policies) it can't change much.

Reply:
It would and there is no guarantee the rule is always be the last, renamed